src/components/Dialog-Window/Tabs/DialogTabs.tsx

```diff
@@ -146,46 +211,44 @@ export const DialogTabs: React.FC<TabProps> = ({ item }) => {
     text: t('dialogTextLabel'),
   };
 
-  const smallScreen = useMedia('(max-width: 768px)');
-
-  const hasNote = item.dialog?.hasNote;
-
-  // Only show tabs if there is more than one item to choose from
-  const tabItemslength = tabItems(item).length;
-  const showTabs = tabItemslength + (hasNote && smallScreen ? 1 : 0) > 1;
+  const dialogNote = (
+    <DialogNote
+      maxLength={item.dialog?.maxWordCount ?? 160}
+      id={item.id}
+      smallScreen
+    />
+  );
 
   return (
     <Root
       className={styles.tabs}
       defaultValue={defaultTabValue(item)}
       orientation="vertical"
     >
-      <List
-        className={showTabs ? styles.list : ''}
-        aria-label={t('dialogTabListAriaLabel')}
-      >
-        {showTabs && tabLabelItems(item, translation)}
-        {smallScreen && hasNote ? (
-          <Trigger key="notes" className={styles.trigger} value="notes">
-            {t('dialogNoteLabel')}
-          </Trigger>
-        ) : null}
-      </List>
+      {showTabs ? (
+        <List
+          className={showTabs ? styles.list : ''}
+          aria-label={t('dialogTabListAriaLabel')}
+        >
+          {showTabs && tabLabelItems(item, translation)}
+          {showNote ? (
+            <Trigger key="notes" className={styles.trigger} value="notes">
+              {t('dialogNoteLabel')}
+            </Trigger>
+          ) : null}
+        </List>
+      ) : null}
       <div
-        className={`${styles.tabItemWrapper} ${
-          !showTabs ? styles.marginTop : ''
+        className={`${styles.tabItemWrapper} ${!showTabs ? styles.marginTop : ''
         }`}
       >
-        {tabItems(item)}
-        {smallScreen && hasNote ? (
+        {dialogContent(item)}
+        {showNote ? (showTabs ? (
           <Content key="notes" value="notes" className={styles.noteWrapper}>
-            <DialogNote
-              maxLength={item.dialog?.maxWordCount ?? 160}
-              id={item.id}
-              smallScreen
-            />
-          </Content>
-        ) : null}
+            {dialogNote}
+          </Content>) : (
+          dialogNote
+        )) : null}
```

Comment:
I suggest removing the nested ternary to improve readability.

```suggestion
        {showNote && (showTabs ? (
          <Content key="notes" value="notes" className={styles.noteWrapper}>
            {dialogNote}
          </Content>) : (
          dialogNote
        ))}
```